server/db/order.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require ('sequelize');
+const db = require('./_db');
+//import join here
+
+const Order = db.define('order', {
+  date: {
+    type: Sequelize.DATEONLY
+  }, 
+  totalCost: {
```

Comment:
everytime a pending order has an item added or deleted we would have to request an operation to update this column. What do you think about removing this attribute, and use some sort of function to do the calculations only when we need to render the total page? Otherwise, we're updating every time even if the total is not needed to be displayed anywhere yet (possibly)

Reply:
Writing virtuals to get totalQuant and totalCost values.